laterpay/application/Controller/Frontend/Invoice.php

```diff
@@ -20,6 +20,9 @@ class LaterPay_Controller_Frontend_Invoice extends LaterPay_Controller_Base
      */
     public function the_invoice_indicator() {
         echo laterpay_sanitized( $this->get_text_view( 'frontend/partials/widget/invoice-indicator' ) );
+
+        wp_enqueue_script( 'laterpay-yui' );
+        wp_enqueue_script( 'laterpay-invoice-indicator' );
```

Comment:
@AliaksandrVahura-ScienceSoft this is not related to the ticket, its another issue we figured yesterday ;)